{"uuid": "c5832411-1774-4366-9c0b-41d1a582167b", "children": ["19c5fead-52a7-4bb4-b614-add26e14d813"], "befores": [{"name": "setup_driver", "status": "broken", "statusDetails": {"message": "TypeError: WebDriver.__init__() got an unexpected keyword argument 'executable_path'\n", "trace": "  File \"C:\\Users\\prade\\PycharmProjects\\allgeo_web_app_pro\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\prade\\PycharmProjects\\allgeo_web_app_pro\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\prade\\PycharmProjects\\allgeo_web_app_pro\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\prade\\PycharmProjects\\allgeo_web_app_pro\\conftest.py\", line 56, in setup_driver\n    web_driver = webdriver.Chrome(\n                 ^^^^^^^^^^^^^^^^^\n"}, "start": 1712746768516, "stop": 1712746768517}], "start": 1712746768516, "stop": 1712746768537}